cf-agent/cf-agent.c

```diff
@@ -324,6 +324,74 @@ int main(int argc, char *argv[])
 /* Level 1                                                         */
 /*******************************************************************/
 
+static void BootstrapConfig(GenericAgentConfig *config, const char *argument)
```

Comment:
Why not just `Bootstrap()`?

Reply:
Bootstrapping is actually done in `GenericAgentDiscoverContext()`, this code just sets up the config correctly for later.

Reply:
Yeah, I saw that in the second commit. What about `ConfigureBootstrap()` then?

Reply:
Fair enough.